tasks/task082_babi_t1_single_supporting_fact_question_generation.json

```diff
@@ -0,0 +1,6053 @@
+        },
+        {
+            "input": "Passage: Mary moved to the bathroom. John went to the hallway. Daniel went back to the hallway. Sandra moved to the garden. John moved to the office. Sandra journeyed to the bathroom. Mary moved to the hallway. Daniel travelled to the office. John went back to the garden. John moved to the bedroom.",
+            "output": "Where is Sandra?",
+            "explanation": "There are two facts mentioning Sandra; 'Sandra moved to the garden.' and 'Sandra journeyed to the bathroom.'. First Sandra went to the garden and then she went to the bathroom. Therefore, we can create a question 'Where is Sandra?' to know where she is currently."
+        }
+    ],
+
+    "Negative Examples": [ 
+        {
+            "input": "Passage: Mary moved to the bathroom. John went to the hallway.",
+            "output": "Which place is Mary at?",
+            "explanation": "This is a bad question as it is not of the form 'Where is <person_name>?'. The correct question would be- 'Where is Mary?'"
+        },
+        {
+            "input": "Passage: Mary moved to the bathroom. John went to the hallway.",
+            "output": "Where is Jack?",
+            "explanation": "Though the question is of form 'Where is <person_name>?', this is a bad example because the question is not answerable from any facts provided. Question must refer to the name of the person that appears in the facts. That way, the question would be answerable. In this case, there are two questions possible; 'Where is Mary?' and 'Where is John?'."
+        }
+    ],
+
+    "Instances": [
+        {
+            "input": "Passage: Mary moved to the bathroom. John went to the hallway.",
+            "output": [
+                "Where is Mary?"
+            ]
+        },
+        {
+            "input": "Passage: Mary moved to the bathroom. John went to the hallway. Daniel went back to the hallway. Sandra moved to the garden.",
+            "output": [
+                "Where is Daniel?"
+            ]
+        },
+        {
+            "input": "Passage: Mary moved to the bathroom. John went to the hallway. Daniel went back to the hallway. Sandra moved to the garden. John moved to the office. Sandra journeyed to the bathroom.",
+            "output": [
+                "Where is Daniel?"
+            ]
+        },
+        {
+            "input": "Passage: Mary moved to the bathroom. John went to the hallway. Daniel went back to the hallway. Sandra moved to the garden. John moved to the office. Sandra journeyed to the bathroom. Mary moved to the hallway. Daniel travelled to the office.",
+            "output": [
+                "Where is Daniel?"
+            ]
+        },
```

Comment:
Is it possible to augment each instance with all the possible outputs? 

For example, in this case, the output should be: 
```
            "output": [
                "Where is Daniel?", "Where is Mary?", "Where is John?", "Where is Daniel?", "Where is Sandra?", ...
            ]```

Reply:
This shouldn't be difficult since the dataset is using a closed set of names, I think.

Reply:
Updated the instances with all possible questions for a given passage in the new commit.

Reply:
Unique question is the way to go. The list really shows all the unique outputs.